modules/ibmz-configure-devices-mco.adoc

```diff
@@ -0,0 +1,263 @@
+// Module included in the following assemblies:
+//
+// * post-installation-configuration/ibmz-post-install.adoc
+
+:_content-type: PROCEDURE
+[id="configure-additional-devices-using-mco_{context}"]
+= Configuring additional devices using the Machine Config Operator (MCO) 
+
+Tasks in this section describe how to use features of the Machine Config Operator to configure additional devices in an {ibmzProductName} or LinuxONE environment. Configuring devices with the MCO is persistent but only allows specific configurations for compute nodes. MCO does not allow control plane nodes to have different configurations.
+
+.Prerequisites
+
+* You are logged in to the cluster as a user with administrative privileges.
+* The device must be available to the z/VM guest.
+* The device is already attached.
+* The device is not included in the `cio_ignore` list, which can be set in the kernel parameters.
+* You have created a `MachineConfig` object file with the following YAML:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: worker0
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,worker0]}
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/worker0: ""
+----
+
+[id="configuring-fcp-host"]
+== Configuring a Fibre Channel Protocol (FCP) host
+
+The following is an example of how to configure an FCP host adapter with N_Port Identifier Virtualization (NPIV) by adding a udev rule.
+
+.Procedure
+
+. Take the following sample udev rule `441-zfcp-host-0.0.8000.rules`:
++
+[source,terminal]
+----
+ACTION=="add", SUBSYSTEM=="ccw", KERNEL=="0.0.8000", DRIVER=="zfcp", GOTO="cfg_zfcp_host_0.0.8000"
+ACTION=="add", SUBSYSTEM=="drivers", KERNEL=="zfcp", TEST=="[ccw/0.0.8000]", GOTO="cfg_zfcp_host_0.0.8000"
+GOTO="end_zfcp_host_0.0.8000"
+
+LABEL="cfg_zfcp_host_0.0.8000"
+ATTR{[ccw/0.0.8000]online}="1"
+
+LABEL="end_zfcp_host_0.0.8000"
+----
+
+. Convert the rule to Base64 encoded, by running the following command:
```

Comment:
Comma could be removed here, eg: 

. Convert the rule to Base64 encoded by running the following command: